frontend/src/features/admin-form/settings/components/EmailFormSection.tsx

```diff
@@ -47,20 +50,30 @@ export const EmailFormSection = ({
 
   useEffect(() => reset({ emails: initialEmails }), [initialEmails, reset])
 
+  const mdComponents = useMdComponents()
+
   return (
-    <FormProvider {...formMethods}>
-      <FormControl isInvalid={!isEmpty(errors)}>
-        <FormLabel
-          isRequired
-          useMarkdownForDescription
-          description={`Add at least **2 recipients** to prevent loss of response. Learn more on [how to guard against email bounces](${GUIDE_PREVENT_EMAIL_BOUNCE}).`}
-        >
-          Emails where responses will be sent
-        </FormLabel>
-        <AdminEmailRecipientsInput onSubmit={handleSubmitEmails} />
-        <FormErrorMessage>{get(errors, 'emails.message')}</FormErrorMessage>
-      </FormControl>
-    </FormProvider>
+    <>
+      <Flex bg="primary.100" p="1rem">
+        <Icon as={BiBulb} color="primary.500" fontSize="1.5rem" mr="0.5rem" />
+        <MarkdownText
+          components={mdComponents}
+        >{`Require routing and approval? [Try using Checkpoint to set up an approval workflow](${OGP_CHECKPOINT})`}</MarkdownText>
+      </Flex>
```

Comment:
Consider refactoring this section to another component (`<CheckpointAdsSection />`) so that it will be easier to remove/shift when the time comes.